MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -218,6 +219,30 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
     }
 }
 
+// MARK: NavigationMapViewDelegate
+
+extension RouteMapViewController: NavigationMapViewDelegate {
+    
+    func navigationMapView(_ mapView: NavigationMapView, shouldUpdateTo location: CLLocation) -> CLLocation? {
```

Comment:
Maybe add `@objc(navigationMapView:shouldUpdateToLocation:)` to help with Objective-C.